### DISCUSSION TITLE
Use native PIP (for real YT Premium users)

### DISCUSSION THREAD

**Discussion by 0evers** [qnblackcat/uYouPlus#260] [Ideas]
I bought YT Premium cheap over Argentina VPN (1,43 USD per month for family) for a better overall compatibility across all my devices.

On my iPad uyou+ works as before, but it would be great if uyou+ provide some settings for these users.
Most useful would be if the native PIP player could be used because it's more stable.

I would be very grateful if this would be implemented.

**Comment by qnblackcat** [comment]
YouPiP has been using  native PiP from the beginning. What exactly do you mean “native PiP player”?
